models/voteModel.js

```diff
@@ -17,7 +17,7 @@ const voteSchema = Schema(
     },
     action: {
       type: String,
-      required: true,
```

Comment:
this field is required, you either vote up or down or don't vote at all

Reply:
Required: false === don't vote at all

You get?